lumicks/pylake/adjustments.py

```diff
@@ -126,7 +126,7 @@ def nothing(cls):
 
 
 def _make_cmap(name, color):
-    LinearSegmentedColormap.from_list(name, colors=[(0, 0, 0), color])
+    return LinearSegmentedColormap.from_list(name, colors=[(0, 0, 0), color])
```

Comment:
😊

Reply:
😒

Reply:
🙈

Reply:
I think the recommendation is there so you actually get a name in call stacks instead of `<lambda>`.

But um, we can add an exception for `E731` if you want :D